source/MRMesh/MRMultiwayAligningTransform.h

```diff
@@ -20,10 +20,13 @@ class MultiwayAligningTransform
     /// initializes internal data to start registering given number of objects
     MRMESH_API explicit MultiwayAligningTransform( int numObjs = 0 );
 
+    MRMESH_API MultiwayAligningTransform( MultiwayAligningTransform&& ) noexcept;
+    MRMESH_API MultiwayAligningTransform& operator=( MultiwayAligningTransform&& ) noexcept;
```

Comment:
why is it needed?

Reply:
Not strictly needed, but making it movable is free, so why not I think.